### PR TITLE
Initialize hasZ in the constructor of GML3

### PR DESCRIPTION
**Pull request by Jenselme**
See https://github.com/openlayers/openlayers/pull/6677#discussion_r116012108

### CODE REVIEW

**Comment by Jenselme** [comment]
@fredj Can you review this?

File: src/ol/format/gml3.js [hunk 1]
```diff
@@ -67,6 +67,13 @@ ol.format.GML3 = function(opt_options) {
   this.schemaLocation = options.schemaLocation ?
       options.schemaLocation : ol.format.GML3.schemaLocation_;
 
+  /**
+   * @protected
```
Comment (bartvde) [inline src/ol/format/gml3.js]: can be @private since there is no class extending GML3

Comment (Jenselme) [inline src/ol/format/gml3.js]: @bartvde Done

**Comment by bartvde** [comment]
@Jenselme it's complaining about the indent

**Comment by Jenselme** [comment]
@bartvde fixed!

**Comment by bartvde** [comment]
thanks

**Pull request merged**